cime_config/cesm/allactive/config_compsets.xml

```diff
@@ -46,37 +46,42 @@
   </compset>
 
   <compset>
-    <alias>B1850RG</alias>
+    <alias>B1850RGWs</alias>
```

Comment:
I'd like to understand the requirement for an out of the box WACCM compset with all active components.